Vault-Deployment/values/vault-values.yaml

```diff
@@ -102,35 +84,13 @@ server:
             leader_client_key_file = "/vault/userconfig/vault-ha-tls/vault-playground-cert.key"
             leader_ca_cert_file = "/vault/userconfig/vault-ha-tls/vault-playground-cert.ca"
           }
-
-          autopilot {
```

Comment:
@Xelef2000 I that used to demonstrate something particular? If not, we might as well remove it.